ads/google/adsense.js

```diff
@@ -41,6 +41,12 @@ export function adsense(global, data) {
   if (data['adHost']) {
     i.setAttribute('data-ad-host', data['adHost']);
   }
+  if (data['adtest']) {
+    i.setAttribute('data-adtest', data['adtest']);
+  }
+  if (data['tagOrigin']) {
+    i.setAttribute('data-tag-origin', data['tagOrigin']);
+  }
```

Comment:
@tdrl I don't see any tests for adsense. Obviously would be nice if there were. They can just go to `test/manual` folder.

Without tests refactoring is a bit riskier. But it's certainly possible to do a simple for-loop on data values and assign them to attributes here. There's a `dashToCamelCase` to help along. Alternatively, you can just put a simple const map in this file that would map expected `data.abCd` to your `data-ab-cd`.